serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/States.kt

```diff
@@ -48,5 +59,11 @@ class MatrixChatRoom(val msession: MatrixSession, val room_id: String, val messa
     fun exitRoom(): MatrixState {
         return MatrixRooms(msession, msession.rooms, "Back to rooms. You can still do a :sync...")
     }
+
+    fun getRoomName(): String {
```

Comment:
same property thought - also, maybe Room should get it's name on creation, so it doesn't have to re-lookup?

Reply:
Saving the room name is definitely the way to go.